src/renderer/qssWidget/js/qssQuickFoldersWidget.js

```diff
@@ -0,0 +1,173 @@
+/**
+ * The QSS Quick folders widget
+ *
+ * Represents the QSS Quick folders widget. Shows input and button and on
+ * submit tries to find if there is a folder with that name (contained in
+ * siteconfig's morphicQuickFolderPath folder) and if there is one opens it.
+ *
+ * Copyright 2019 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ * The research leading to these results has received funding from the European Union's
+ * Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+/* global fluid */
+
+"use strict";
+(function (fluid) {
+    var gpii = fluid.registerNamespace("gpii");
+
+    /**
+     * QSS Quick folders widget
+     */
+    fluid.defaults("gpii.qssWidget.quickFolders", {
+        gradeNames: ["fluid.viewComponent", "gpii.psp.selectorsTextRenderer"],
+
+        selectors: {
+            folderSearch: ".flc-qssQuickFoldersWidget-folderSearch",
+            settingTitle: ".flc-qssSearchWidget-settingTitle",
+            searchField: ".flc-search",
+            searchButton: ".flc-searchButton",
+            labelTitle: ".flc-qssSearchWidget-label",
+            errorMessage: ".flc-qssSearchWidget-errorMessage"
+        },
+
+        enableRichText: true,
+
+        model: {
+            setting: {},
+            value: "{that}.model.setting.value",
+            messages: {
+                labelTitle: "{quickFolders}.model.messages.label",
+                searchButtonLabel: "{quickFolders}.model.messages.searchButtonLabel",
+                alertLabel: "{quickFolders}.model.messages.alertLabel"
+            },
+            errorMessageSelector: "{that}.options.selectors.errorMessage",
+        },
+
+        listeners: {
+            onCreate: {
+                func: "{that}.onCreate"
```

Comment:
No need for a single-purpose invoker here - better is to just write out the invocation inline as a namespaced listener - 
````
listeners: {
    "onCreate.hideErrorMessage": {
        func: "{gpii.qssWidget.quickFolders}.errorMessage.onSuccess.fire"
    }
}